pyne/tally.pyx

```diff
@@ -21,6 +24,32 @@ from pyne.utils import QAWarning
 
 warn(__name__ + " is not yet QA compliant.", QAWarning)
 
+cdef vector[int] to_vector_int(value):
+    cdef vector[int] value_proxy
+    cdef int ivalue
+    cdef int value_size
+    cdef int * value_data
+    
+    value_size = len(value)
+    value_proxy = vector[int](< size_t > value_size)
+    for ivalue in range(value_size):
+        value_proxy[ivalue] = <int > value[ivalue]
+    return value_proxy
+
+
+cdef vector[double] to_vector_double(value):
+    cdef vector[double] value_proxy
+    cdef int ivalue
+    cdef int value_size
+    cdef double * value_data
+    
+    value_size = len(value)
+    value_proxy = vector[double](< size_t > value_size)
+    for ivalue in range(value_size):
+        value_proxy[ivalue] = <double > value[ivalue]
+    return value_proxy
```

Comment:
@gonuke this simplified version seems to be working with python3, let's check if it works with python2 as well...

Reply:
Is it being tested over a broad enough range of use cases?  It seems like something we could throw a lot of tests at without much effort.